src/asm3/dbupdate.py

```diff
@@ -2666,6 +2718,8 @@ def user(tid: int, username: str, realname: str, password: str, superuser: bool)
     sql += lookup2("stockusagetype", "UsageTypeName", 5, _("Sold", l))
     sql += lookup2("stockusagetype", "UsageTypeName", 6, _("Stocktake", l))
     sql += lookup2("stockusagetype", "UsageTypeName", 7, _("Wasted", l))
+    sql += lookup2("stockusagetype", "UsageTypeName", 8, _("Movement", l))
+    sql += taxrate(1, _("Tax Free", l), 0.0)
```

Comment:
we should use your new tax rate table on payments in future, that can be a new ticket. I'm just putting a comment here to remind us to do it :)